spec/factories/subjects/primary_subjects.rb

```diff
@@ -0,0 +1,63 @@
+FactoryBot.define do
+  subjects = [
+     { subject_name: "Primary", subject_code: "00" },
+     { subject_name: "Primary with English", subject_code: "01" },
+     { subject_name: "Primary with geography and history", subject_code: "02" },
+     { subject_name: "Primary with mathematics", subject_code: "03" },
+     { subject_name: "Primary with modern languages", subject_code: "04" },
+     { subject_name: "Primary with physical education", subject_code: "06" },
+     { subject_name: "Primary with science", subject_code: "07" },
+   ]
```

Comment:
Saw this somewhere else, feels like a map would better fit here:

```suggestion
  subjects = {
    "Primary"                            => "00",
    "Primary with English"               => "01",
    "Primary with geography and history" => "02",
    "Primary with mathematics"           => "03",
    "Primary with modern languages"      => "04",
    "Primary with physical education"    => "06",
    "Primary with science"               => "07",
  }

```